nle/tests/test_nethack.py

```diff
@@ -89,6 +89,21 @@ def test_permonst(self):
         mon = nethack.permonst(1)
         self.assertEqual(mon.mname, "killer bee")
 
+        # Test https://github.com/pybind/pybind11/issues/2394
+        for _ in range(2):
+            mon = nethack.permonst(0)
+            self.assertEqual(mon.mname, "giant ant")
+
+    def test_illegal_numbers(self):
+        with self.assertRaisesRegex(IndexError, "NUMMONS"):
+            nethack.permonst(10000)
+
+        with self.assertRaisesRegex(IndexError, "NUMMONS"):
+            nethack.permonst(-1)
+
+        with self.assertRaisesRegex(IndexError, "MAXMCLASSES"):
```

Comment:
Just to be consistent, I would probably add an exception check for negative values for `from_mlet`.

Reply:
I would if I could -- but mlet is a char which in Python gets mapped to a string of length 1. I cannot produce negative values there :D